diff --git a/CMakeLists.txt b/CMakeLists.txt
index 9efd451fb..05066be82 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -643,26 +643,6 @@ target_optional_compile_definitions(torrent-rasterbar PUBLIC FEATURE NAME mutabl
 target_optional_compile_definitions(torrent-rasterbar PUBLIC FEATURE NAME streaming DEFAULT ON
 	DESCRIPTION "Enables support for piece deadline" DISABLED TORRENT_DISABLE_STREAMING)
 

 find_public_dependency(OpenSSL)
 set_package_properties(OpenSSL
 	PROPERTIES
